tests/test_recordio.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from dcos import recordio
+from dcos.errors import DCOSException
+
+
+def test_encode():
+    try:
+        encoder = recordio.Encoder(json.dumps)
```

Comment:
Your're right, I need to fix this. They should be an array of bytes. They are only strings in this one case because I am using a serializer that converts input to a string.